### PR TITLE
Create performance_review.md

### PR DESCRIPTION
**Pull request by jain-amit**

### CODE REVIEW

File: articles/_drafts/performance_review.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+##Overview
+* Follow prescribed format
+* Feedback should have been given throughout review period so there should not be any surprises in review
+* Provide specific examples for both positive and negative behaviour
+* The majority of time writing a review is taken to ensure the intent comes across not to be lost by the words used
+* Areas to work on for the period
+  * Should be already something you have discussed as part of their career growth
+  * Should be specific and meaningful for their career
+  * Should have at least one metric to measure
+  * You can structure goals using SMART
```
Comment (frankywahl) [inline articles/_drafts/performance_review.md]: You can put a link to smart like so: 

```
[SMART](https://en.wikipedia.org/wiki/SMART_criteria#Developing_SMART_goals)
```

Then remove the bullet point that is too nested

File: articles/_drafts/performance_review.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    * https://en.wikipedia.org/wiki/SMART_criteria#Developing_SMART_goals
+* Have a peer review what you have written update accordingly
+*	Provide as a PDF prior to meeting so that individual can review
```
Comment (frankywahl) [inline articles/_drafts/performance_review.md]: Extra space?

File: articles/_drafts/performance_review.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+* Feedback should have been given throughout review period so there should not be any surprises in review
+* Provide specific examples for both positive and negative behaviour
+* The majority of time writing a review is taken to ensure the intent comes across not to be lost by the words used
+* Areas to work on for the period
+  * Should be already something you have discussed as part of their career growth
+  * Should be specific and meaningful for their career
+  * Should have at least one metric to measure
+  * You can structure goals using SMART
+    * https://en.wikipedia.org/wiki/SMART_criteria#Developing_SMART_goals
+* Have a peer review what you have written update accordingly
```
Comment (frankywahl) [inline articles/_drafts/performance_review.md]: missing keyword `and`?

File: articles/_drafts/performance_review.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+#Performance Review
```
Comment (frankywahl) [inline articles/_drafts/performance_review.md]: Personal consistency preference in title format
<POUND><SPACE><TITLE>

You're missing the space. 

I can take care of it if you want when I go publishing it live.

Comment (jain-amit) [inline articles/_drafts/performance_review.md]: Please do take care of it before publishing

**Pull request merged**